This patch is for festival for Debian GNU/Linux, to correct
spelling errors found by Lintian

diff -Naur b/src/arch/festival/wave.cc c/src/arch/festival/wave.cc
--- b/src/arch/festival/wave.cc	2011-09-14 20:56:08.000000000 +1000
+++ c/src/arch/festival/wave.cc	2011-09-14 21:06:55.000000000 +1000
@@ -597,7 +597,7 @@
 
     init_subr_4("wave.save",wave_save,
  "(wave.save WAVE FILENAME FILETYPE SAMPLETYPE)\n\
-  Save WAVE in FILENAME, respecting FILETYPE and SAMPLETYPE if specifed\n\
+  Save WAVE in FILENAME, respecting FILETYPE and SAMPLETYPE if specified\n\
   if these last two arguments are unspecified the global parameters\n\
   Wavefiletype and Wavesampletype are used.  Returns t is successful\n\
   and throws an error if not.");  
diff -Naur b/src/modules/MultiSyn/UnitSelection.cc c/src/modules/MultiSyn/UnitSelection.cc
--- b/src/modules/MultiSyn/UnitSelection.cc	2011-09-14 20:55:48.000000000 +1000
+++ c/src/modules/MultiSyn/UnitSelection.cc	2011-09-14 21:07:49.000000000 +1000
@@ -468,7 +468,7 @@
 	CHECK_PTR(jc);
       }
       else
-	EST_error( "du_voice_setJoinCost: currently t is the only supported second arguement" );
+	EST_error( "du_voice_setJoinCost: currently t is the only supported second argument" );
       
       duv->setJoinCost(jc,true);
     }
